bigbluebutton-html5/imports/ui/components/video-provider/component.tsx

```
@@ -1107,7 +1108,7 @@ class VideoProvider extends Component<VideoProviderProps, VideoProviderState> {
         role,
       },
     }, 'Local camera stream stopped unexpectedly');
-
+    throw new Error('WebRTC peer failed');
     const error = new Error('inactiveError');
     this.onWebRTCError(error, stream, isLocal);
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)